Makefile

```diff
@@ -118,6 +119,7 @@ test-sweep: ## Clean up test tenant
 dev-up: ## Bootstrap the development containers
 	${call print, "Starting development containers"}
 	@docker-compose up -d
+	@docker-compose exec -- auth0 curl -4 --retry-connrefused --retry 15 --retry-delay 2 -sS -o /dev/null -X POST http://localhost:8080/__admin/mappings/reset
```

Comment:
Now that these files are exposed as volumes, you can simply edit the files locally and when you rerun the tests they will automatically be synced.